application/controllers/admin/globalsettings.php

```diff
@@ -72,8 +72,9 @@ public function sendTestEmail()
 
         $sFrom = Yii::app()->getConfig("siteadminname")." <".Yii::app()->getConfig("siteadminemail").">";
         
-        $sSubject = gT('User data');
         $sSiteName = Yii::app()->getConfig('sitename');
+        $sSubject = gT('Test email from ') .$sSiteName;
```

Comment:
Maybe best to `sprintf(gT('Test email from : %s'),Yii::app()->getConfig('sitename'))` (and we surely must filter sitename) :)

Reply:
Do you think it might be better if the code just didn't use the sitename in the subject at all? Maybe the subject should just be "Test email from LimeSurvey"?

Reply:
I vote for using sitename from config.

Reply:
:+1: